lib/cartopy/tests/test_crs.py

```diff
@@ -36,7 +36,7 @@ def test_osni(self, approx):
         ll = ccrs.Geodetic()
 
         # results obtained by nearby.org.uk.
-        lat, lon = np.array([54.5622169298669, -5.54159863617957],
+        lon, lat = np.array([-5.54159863617957, 54.5622169298669],
```

Comment:
Why is this even put through `np.array` in the first place if it's split immediately?

Reply:
🤷 I was wondering the same thing, but really didn't want to go refactoring more than this PR title suggests.